product_docs/docs/pem/8/supported_platforms.mdx

```diff
@@ -8,4 +8,14 @@ For information about the platforms and versions supported by PEM, see [Platform
 !!! Note
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
-## Database compatibility
\ No newline at end of file
+## Database compatibility
+
+The following table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD) for PEM 8.x.
+
+|           |**PEM 8.x**                   |                    |
+|:----------|:-----------------------------|:-------------------|
+|           |**As a monitored instance**   |**As a backend**    |
+|**PG**     |11, 12, 13, 14                |11, 12, 13, 14      |
+|**PGE**    |11, 12, 13, 14                |11, 12, 13, 14      |
+|**EPAS**   |11, 12, 13, 14                |11, 12, 13, 14      |
```

Comment:
I think we can take EPAS in first row and then in 2nd and 3rd row PG and PGE respectively

Reply:
Done